app/src/main/java/com/example/tbuckley/brakes/MainActivity.java

```diff
@@ -44,13 +76,21 @@ public void onPageFinished(WebView view, String url) {
                     mClearHistory = false;
                     mWebView.clearHistory();
                 }
+                mProgressBar.setVisibility(View.GONE);
```

Comment:
Should the progress bar be hidden in onPageFinished instead of onPageStarted?

Reply:
This is in onPageFinished.

Reply:
Whoops, diff made it look like onPageStarted...